project/Dependencies.scala

```diff
@@ -163,6 +163,13 @@ object Dependencies {
     )
   )
 
+  val Bmcs = Seq(
+    libraryDependencies ++= Seq(
+      "org.bouncycastle" % "bcpkix-jdk15on" % "1.52",
```

Comment:
Comment with license

Reply:
MIT license. 
https://www.bouncycastle.org/licence.html

Reply:
As in add a comment to the right of dependency saying that, thanks :)